Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -357,6 +358,8 @@ export interface QTestArguments extends Transformer.RunnerArguments {
     qTestEnvironmentVariables?: Transformer.EnvironmentVariable[];
     /** Specify the path relative to enlistment root of the sources from which the test target is built */
     testSourceDir?: RelativePath;
+    /** Path to a file which contains a list of target file names execluded for code coverage instrument */

```

Comment:
>instrument [](start = 94, length = 10)

nit - remove "instrument" or maybe call it "processing" since this new code coverage mechanism used by qtest doesn't do an offline file instrumentation. We call it instrumentation out of habit but technically that's not correct :grinning:

Reply:
nit: typo "execluded", s/for code coverage instrument/for code coverage instrumentation